### PR TITLE
dataclasses installation fix

### PR DESCRIPTION
**Pull request by aby2s**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "lenses",
     "pyyaml",
     "jsonschema",
-    "dataclasses",  # for 3.6, as it ships with 3.7
+    'dataclasses;python_version=="3.6"',  # for 3.6, as it ships with 3.7
```
Comment (ksaaskil) [inline setup.py]: Did you check somehow this is the exact correct syntax? So it doesn't break for "3.6.1"?

Comment (aby2s) [reply setup.py]: I checked that it didn't break the setup procedure in general but it's a good question.

Comment (aby2s) [reply setup.py]: Now it is correct

**Comment by fornwall** [comment]
👍

**Pull request merged**